R/settings.R

```diff
@@ -49,18 +38,41 @@ read_settings <- function(filename) {
   }
 }
 
+read_config_file <- function(config_file) {
+  if (is.null(config_file)) {
+    return(NULL)
+  }
+
+  config <- new.env()
+  if (endsWith(config_file, ".R")) {
```

Comment:
This check should be upgraded later down the line if we stick with R config files.